src/transformers/models/mpnet/tokenization_mpnet.py

```diff
@@ -0,0 +1,194 @@
+# coding=utf-8
+# Copyright 2018 The HuggingFace Inc. team, Microsoft Corporation.
+# Copyright (c) 2018, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tokenization classes for MPNet."""
+
+
+from typing import List, Optional
+
+from ...tokenization_utils import AddedToken
+from ...utils import logging
+from ..bert.tokenization_bert import BertTokenizer
+
+
+logger = logging.get_logger(__name__)
+
+VOCAB_FILES_NAMES = {"vocab_file": "vocab.txt"}
+
+PRETRAINED_VOCAB_FILES_MAP = {
+    "vocab_file": {
+        "mpnet-base": "",
+    }
+}
+
+PRETRAINED_POSITIONAL_EMBEDDINGS_SIZES = {
+    "mpnet-base": 512,
+}
+
+PRETRAINED_INIT_CONFIGURATION = {
+    "mpnet-base": {"do_lower_case": True},
+}
+
+
+class MPNetTokenizer(BertTokenizer):
```

Comment:
Colud we maybe try to not inherit from `BertTokenizer` here? It seems like this class overwrites quite some methods - in this case it would be awesome if you could just copy past the required `BertTokenizer` functionality here. @LysandreJik @sgugger - what is your opinion on that?

Reply:
Agreed, I don't see the point of inheriting from `BertTokenizer` here either.

Reply:
@patrickvonplaten I have removed inheritance and updated [tokenization_mpnet.py](https://github.com/StillKeepTry/transformers/blob/master/src/transformers/models/mpnet/tokenization_mpnet.py) and [tokenization_mpnet_fast.py](https://github.com/StillKeepTry/transformers/blob/master/src/transformers/models/mpnet/tokenization_mpnet_fast.py).  Can you have a check again?